examples/client/Locomotion/src/Components/PageHeader/styled.js

```diff
@@ -26,8 +26,8 @@ export const HeaderIconContainer = styled(Button)(({ side, theme }) => `
 `);
 
 export const HeaderIcon = styled.Image`
-  width: ${({ width }) => (width || '18px')};
-  height: ${({ height }) => (height || '18px')};
+  width: ${({ width }) => (width || 18)};
+  height: ${({ height }) => (height || 18)};
```

Comment:
in general check if we need these fixes to more csses

Reply:
it was error while the build started, we do have more errors in the build (key is not a prop, keys are missing) 
i think treating them on another branch